Library/Formula/netperf.rb

```diff
@@ -11,6 +11,12 @@ class Netperf < Formula
     sha256 "4e0aa192670d1de341e77e0bd3af4b2cccc6a5b12a734e922e92c7b7d50f281d" => :mountain_lion
   end
 
+  # patch fixes demo mode which is required by a lot of software that depends on netperf
+  patch :p2 do
+    url "https://raw.githubusercontent.com/trelane/netperf-brew-patch/master/netlib.diff"
```

Comment:
I have written a blog article on bufferbloat for Zenoss ($employer).  In the process of the article I found and fixed this bug.  Flent (www.flent.org) can't run without netperf compiled with demo mode enabled.  I will be covering Flent in the article, so I do expect the formula to get some use.